docs/gathered-data.md

```diff
@@ -382,6 +382,13 @@ Relevant OpenShift API docs:
 Location in archive: config/securitycontentconstraint/, config/clusterrolebinding/
 
 
+## SAPDatahubs
+
+collects `datahubs.installers.datahub.sap.com` resources from SAP/SDI clusters.
+
+Location in archive: config/installers.datahub.sap.com/datahubs/<namespace>/<name>.json
```

Comment:
Hmm we don't have any "third party" custom resources in the archive yet right? Just thinking if it's reasonable path...
Maybe `config/customresources/installers.datahub.sap.com/....` if we have more in the future.

Reply:
I didn't want the path to be longer than what was strictly necessary because it is already a real pain to type. When we discussed the possibilities, this option seemed sufficiently specific that no one would confuse it. In fact, I would say that it would be best if all resources were stored in this format. I'd like to believe that in the future we are going to collect a lot more of these non-Kubernetes, non-OpenShift resources and I feel like it's a bit odd to put them into one directory even though they are not related. I can change it if you want, but it feels like the path is way too long then.

Reply:
I don't have a strong opinion TBH. I just wanted to discuss it. I think the length of the path doesn't play any role. I know that the structure of the IO archive is not ideal but I would like to keep it as clear as possible. 

My feeling is that we're blindly putting everything under `config`. What about putting this CR to the root dir then? Just an idea...

Reply:
I was thinking the same, but we've been putting stuff into `config/` for so long I was not brave enough to suggest putting anything elsewhere. I would say that resource that are gathered verbatim, without any preprocessing (except for potential anonymization where applicable) should be stored in a more formal structure. Either directly in the root of the archive or in a directory like `resources/` or something. Then I'd suggest to use the most sensible directory structure based on the API group and resource name (API version should not be necessary, hopefully), followed by an optional namespace for non-cluster-wide resources and the name of the resource itself. This kind of a standardization, at least for non-core resources (like pods or nodes), should help us with organization of the archive in a more reasonable way than just throwing everything in `config/`. I'll bring this up on Monday's stand-up if I don't forget about it.